jobbergateapi2/apps/application_permissions/models.py

```diff
@@ -0,0 +1,14 @@
+"""
+Database model for the ApplicationPermission.
+"""
+from sqlalchemy import Integer, String, Table
+from sqlalchemy.sql.schema import Column
+
+from jobbergateapi2.metadata import metadata
+
+application_permissions_table = Table(
+    "application_permissions",
+    metadata,
+    Column("id", Integer, primary_key=True),
```

Comment:
It is very common to use `id` for database id and it doesn't will be a problem in any moment because it is a "property" of some object. Actually it is better to use id here, than some other, since the user expects the table to have a column id.